Makefile

```diff
@@ -1,21 +1,41 @@
-.DEFAULT_GOAL:= all
+.DEFAULT_GOAL := all
 
-.PHONY: all
-all: vet test
+all: test lint finish
```

Comment:
why?

Reply:
Because in the Github action yaml he is already doing it this way and
for me it makes more sense to first check the linting before I start with
the unit/integration tests, i.e. fail early.

Reply:
I reordered it. Now it matches also the order of the steps in the github pipeline.
In the build job there is the build and linting.
In the test job there are the unit and integration tests.

In general I agree with @frank-e-zimmer here, first check the code structure and then test the code.

Reply:
done